### ISSUE TITLE
(Voiceovers) Unable to select 'none' for voiceover once accepted

### ISSUE THREAD

**Issue by whitman** [authorly/interapptive-web#911]
Once a voiceover has been selected, the modal's "Accept" button won't submit when a "None" is selected. Instead, a message comes up from `app.vent.trigger 'show:message', 'msg'`
